pom.xml

```diff
@@ -149,6 +150,17 @@
     </plugins>
   </reporting>
   <profiles>
+  <profile>
+    <id>m2e</id>
```

Comment:
I think @kishorekasi might have been doing this to detect the os within Eclipse.. It shouldn't be needed anymore, since we depend on netty-all